controllers/greptimedbcluster/deployers/meta.go

```diff
@@ -247,6 +247,10 @@ func (b *metaBuilder) BuildDeployment() deployer.Builder {
 				},
 			},
 			Template: *b.generatePodTemplateSpec(),
+			Strategy: appsv1.DeploymentStrategy{
+				Type:          appsv1.RollingUpdateDeploymentStrategyType,
+				RollingUpdate: b.Cluster.Spec.Meta.RollingUpdate,
```

Comment:
Can we support deployment update strategy type of Recreate?
https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy

Reply:
We don't need `Recreate`. In our scenario, `RollingUpdate` is enough.